OWNERS

```diff
@@ -0,0 +1,10 @@
+reviewers:
+  - dlorenc
+  - aaron-prindle
+  - r2d4
+approvers:
+  - dlorenc
+  - luxas
+  - aaron-prindle
+  - jimmidyson
+  - r2d4
```

Comment:
other approvers you might want to add from kubernetes/minikube-admins:
- vishh
- pwittrock

other maintainers you might want to add:
- ethernetdan

other collaborators you might want to add:
- gbraad
- budhrg

Reply:
@spiffxp can you share who is part of kubernetes/minikube-admins?

I understand that you used kubernetes org membership to determine some of it, but it might make more sense for the current maintainers to just apply for membership rather than add people who aren't involved in the project but happen to be members of the org.

Reply:
that was my thought as well, re: asking collaborators to join as members (I'm happy to act as a sponsor if you cc me)

github teams are currently) orthogonal to the contents of OWNERS files, I was just planning on using them as the start of the OWNERS file, so I'm happy to just say this LGTM and you can iterate as needed

minikube-admins:
- @dlorenc 
- @luxas 
- @vishh 
- @pwittrock 
minikube-maintainers:
- @spiffxp (I will remove myself once done helping out here)
- @jimmidyson 
- @ethernetdan 
collaborators with write access (these folks should apply for membership)
- @gbraad 
- @r2d4 
- @budhrg 
- @aaron-prindle 
- @minikube-bot